vt.c: add missing include to fix building with uClibc

Backporting an upstream patch to fix a building issue with uClibc.

Upstream commit:

  https://github.com/kernelslacker/trinity/commit/fb4a1adc4540f0702b84aa900f2b8ebed004885d

Signed-off-by: Vicente Olivert Riera <Vincent.Riera@imgtec.com>

From fb4a1adc4540f0702b84aa900f2b8ebed004885d Mon Sep 17 00:00:00 2001
From: Vicente Olivert Riera <Vincent.Riera@imgtec.com>
Date: Wed, 18 Mar 2015 09:58:46 +0000
Subject: [PATCH] vt.c: add missing include to fix building with uClibc

"#include <linux/serial.h>" is needed in order to build trinity
successfully with uClibc. Otherwise it would fail displaying an error
message like this one:

ioctls/vt.c:152:8: error: invalid application of 'sizeof' to incomplete
type 'struct serial_rs485'
  IOCTL(TIOCGRS485),

ioctls/vt.c:155:8: error: invalid application of 'sizeof' to incomplete
type 'struct serial_rs485'
  IOCTL(TIOCSRS485),

Signed-off-by: Vicente Olivert Riera <Vincent.Riera@imgtec.com>
---
 ioctls/vt.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/ioctls/vt.c b/ioctls/vt.c
index 5e7e7d7..a4a7e0c 100644
--- a/ioctls/vt.c
+++ b/ioctls/vt.c
@@ -1,6 +1,7 @@
 #include <sys/vt.h>
 #include <sys/ioctl.h>
 #include <linux/kd.h>
+#include <linux/serial.h>
 
 #include "utils.h"
 #include "ioctls.h"
-- 
1.7.1

